go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -518,13 +519,26 @@ func (erp *EmergencyReparenter) reparentReplicas(
 	rec := concurrency.AllErrorRecorder{}
 
 	handlePrimary := func(alias string, tablet *topodatapb.Tablet) error {
-		position, err := erp.tmc.PrimaryPosition(replCtx, tablet)
-		if err != nil {
-			return err
-		}
-		if populateReparentJournal {
+		if !intermediateReparent {
+			var position string
+			var err error
+			if ev.ShardInfo.PrimaryAlias == nil {
+				erp.logger.Infof("setting up %v as new primary for an uninitialized cluster", alias)
```

Comment:
Log messages are much nicer IMO when using `topoproto.TabletAliasString`

Reply:
We call `handlePrimary` with `TabletAliasString`, so the alias we're using for the message is well formatted.

Reply:
This is the handlePrimary call - `handlePrimary(topoproto.TabletAliasString(newPrimaryTablet.Alias), newPrimaryTablet)`